test/project_types/theme/tasks/ensure_themekit_installed_test.rb

```diff
@@ -1,4 +1,3 @@
-# frozen_string_literal: true
```

Comment:
Was this removed because of `force_encoding` ?

Reply:
Yes it was - do you know if there's a better way to go about this?

Reply:
I *think* you can call `dup` on the original string which will give non-frozen string which you can then call `force_encoding` on.